mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/AttachmentMapperTest.java

```diff
@@ -136,4 +141,44 @@ public void getAttachmentsShouldReturnTheAttachmentsWhenSome() throws Exception
         //Then
         assertThat(attachments).contains(expected, expected2);
     }
+
+    @Test
+    public void getOwnerMessageIdsShouldReturnEmptyWhenNone() throws Exception {
+        Collection<MessageId> messageIds = attachmentMapper.getOwnerMessageIds(UNKNOWN_ATTACHMENT_ID);
+
+        assertThat(messageIds).isEmpty();
+    }
+
+    @Test
+    public void getOwnerMessageIdsShouldReturnEmptyWhenStoredWithoutMessageId() throws Exception {
+        //Given
+        Attachment attachment = Attachment.builder()
+                .bytes("payload".getBytes(Charsets.UTF_8))
+                .type("content")
+                .build();
+        AttachmentId attachmentId = attachment.getAttachmentId();
+        attachmentMapper.storeAttachment(attachment);
+        
+        //When
+        Collection<MessageId> messageIds = attachmentMapper.getOwnerMessageIds(attachmentId);
+        //Then
+        assertThat(messageIds).isEmpty();
+    }
+
+    @Test
+    public void getOwnerMessageIdsShouldReturnMessageIdWhenStoredWithMessageId() throws Exception {
```

Comment:
Can you demonstrate an attachment can be attached to two message?

Reply:
of course, I missed that in the TODO :)

Reply:
done plus other basic tests